doc/source/conf.py

```diff
@@ -32,8 +32,8 @@
 # ones.
 extensions = [
     'sphinx.ext.autodoc',
-	'numpydoc',
-	'sphinx.ext.imgmath',
+    'numpydoc',
```

Comment:
Thanks.  I upgraded my editor a while back, and I didn't realize that the default setting was no longer converting tabs to spaces.